config/services/nmea-0183.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>nmea-0183</short>
```

Comment:
nmea-0183 is a serial communications protocol so I don't follow what it has to do with firewalld. Are you trying to open up ports for a server that uses nmea-0183? If so use the name of the server/daemon/product. nmea-0183 is nonsensical to firewalld.

Reply:
This is used for servers sharing nmea-0183 data on the network (for sharing one GPS location or time devices with multiple clients, or for submitting sensor data)